docs/source/additional_features.rst

```diff
@@ -177,6 +171,29 @@ commit id of the merge base produced by the git command above. The
 script will decompress the data so that mypy will start with a fresh
 ``.mypy_cache``. Finally, the script runs mypy normally. And that's all!
 
+Caching with mypy daemon
+========================
+
+You can use the :ref:`mypy daemon <mypy_daemon>` for faster
+incremental mypy runs, and it also supports remote caching. The remote
+cache will significantly speed up the the first ``dmypy check`` run
+after starting or restarting the daemon.
+
+The mypy daemon requires extra fine-grained dependency data in
+the cache files which aren't included by default. To use caching with
+the mypy daemon, use the ``--cache-fine-grained`` option in your CI
+build::
```

Comment:
Explain what this flag does (it writes extra information for the daemon to the cache).

Reply:
Added an explanation.